releasenotes-dca/notes/node-unschedulable-4fa23bb955163a3a.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Node schedulability is now a dedicated tag on kubernetes node resources.
```

Comment:
not sure if needed: we may want to note that with this change we also removed `node_status:schedulingdisabled`.

Reply:
sounds a bit too specific for the release note IMHO

Reply:
makes sense 👍 . Just wasn't sure in case a customer is confused that he/she lost the tag 🤔